tools/xchain-suite/src/pages/SwapPage.tsx

```
@@ -73,7 +73,7 @@ export default function SwapPage() {
   // Tracking state
   const [isTrackingOpen, setIsTrackingOpen] = useState(false)
   const [trackingTxHash, setTrackingTxHash] = useState<string | null>(null)
-  const [trackingProtocol, setTrackingProtocol] = useState<'Thorchain' | 'Mayachain' | 'Chainflip'>('Thorchain')
+  const [trackingProtocol, setTrackingProtocol] = useState<'Thorchain' | 'Mayachain' | 'Chainflip' | 'OneClick'>('Thorchain')
```

ESLint

[error] Replace `'Thorchain'` with `⏎····'Thorchain',⏎··`

(prettier/prettier)